yomi/simple_eval.py

```diff
@@ -1,30 +1,56 @@
 """ Simple job that reads from a storage bucket """
 
 import os
-import apiclient.discovery
-import httplib2
+import re
 import json
-from oauth2client.service_account import ServiceAccountCredentials
+from google.cloud import storage
+from google.oauth2 import service_account
 
 SERVICE_ACCOUNT_KEY_LOCATION = os.environ['SERVICE_ACCOUNT_KEY_LOCATION']
 BUCKET_NAME = os.environ['BUCKET_NAME']
 
+MODEL_DIR = 'models'
+EVALUATION_DIR = 'evaluations'
+
 
 def run():
-    """ Talk to GCS. Do some things. """
-    google_api_secret = json.load(open(SERVICE_ACCOUNT_KEY_LOCATION))
+    """ Get the models from GCS and then have them play eachother. """
+    credentials = service_account.Credentials.from_service_account_file(
+        SERVICE_ACCOUNT_KEY_LOCATION)
+    scoped_credentials = credentials.with_scopes(
+        ['https://www.googleapis.com/auth/cloud-platform'])
+
+    # Use the hand-crafted GCS client
+    storage_client = storage.Client(credentials=credentials)
+    bucket = storage_client.get_bucket(BUCKET_NAME)
+    blobs = bucket.list_blobs(prefix=MODEL_DIR)
```

Comment:
if you just look for *.index or *.meta you'll only get one file per model number